mail/api_test.py

```diff
@@ -101,6 +109,27 @@ def test_send_bcc(self, sender_name, mock_post):
         else:
             self.assertEqual(called_kwargs['data']['from'], settings.EMAIL_SUPPORT)
 
+    @data(True, False)
+    def test_send_bcc_error(self, raise_for_status, mock_post):  # pylint: disable=unused-argument
+        """
+        Test that send_bcc raises an exception at the right time
+        """
+        mock_post.return_value = Mock(
+            spec=Response,
+            status_code=HTTP_400_BAD_REQUEST,
+            json=mocked_json()
+        )
+
+        response = MailgunClient.send_bcc(
+            'email subject',
+            'email body',
+            ['a@example.com', 'b@example.com'],
+            raise_for_status=raise_for_status,
+        )
+
+        assert response.raise_for_status.called is raise_for_status
```

Comment:
It's testing that `response.raise_for_status` is called when it should be, which in turn raises an exception. I can change the test description but I think this test is meaningful as is. It says that if `send_bcc` is called and it gets a 400 response, it will call `raise_for_status` and pass along the 400 status code.